scripts/create-package/utils.test.ts

```diff
@@ -90,7 +91,9 @@ describe('create-package/utils', () => {
         nodeVersions: '>=18.0.0',
       };
 
-      (fs.existsSync as jest.Mock).mockReturnValueOnce(false);
+      (fs.promises.stat as jest.Mock).mockReturnValueOnce({
+        isDirectory: jest.fn().mockReturnValueOnce(false),
```

Comment:
Nit: in these two examples, we aren't using the inner mock

```suggestion
        isDirectory: false,
```

Reply:
I believe this is fixed now.